packages/ui-react/src/components/VideoMetaData/__snapshots__/VideoMetaData.test.tsx.snap

```diff
@@ -0,0 +1,27 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`<VideoMetaData> > renders and matches snapshot 1`] = `
+<div>
+  <span>
+    2023
+  </span>
+  <span
+    aria-hidden="true"
+    class="_separator_1267df"
+  >
+    |
+  </span>
+  <span>
+    10m
```

Comment:
Is this read by screenreaders correctly (10 meters versus 10 minutes)?

Reply:
Well thought of. Indeed it is read as meters instead of minutes.
I will fix it, but I also need to take translations in to consideration then.

Reply:
I changed "m" to "min" and "h" to "hrs". According to ChatGTP the Spanish abbreviations for minutes is the same as english, while the abbreviation of hours is just "h". There are a lot more of these small translation error within the app, which I suggest we should fix in one go. A little to much effort for now, I think.